src/common/services/request/query/fetch-testimonials.js

```diff
@@ -0,0 +1,98 @@
+export const fetchFiltered = {
```

Comment:
Rename this file as it has both fetch and mutations.

Reply:
The parent folder of `fetch-testimonials.js` is query. Should the mutation exist in the same file?